.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -20,8 +20,7 @@ If multiple identifiers make sense you can also state the commands multiple time
 -->
 /area TODO
 /kind bug
-/priority normal
-/platform alicloud
+/priority normaln/platform alicloud
```

Comment:
You could revert similar changes.

See https://github.com/gardener/gardener-extension-os-suse-chost/pull/22#discussion_r452649842.

Reply:
done